SSA/Projects/InstCombine/ForStd.lean

```diff
@@ -2,8 +2,11 @@ import Std.Data.BitVec
 
 open Std
 
-def ofBool : (Bool) -> Std.BitVec 1
- | c => if c then 1 else 0
+namespace Std.BitVec
+
+def ofBool : Bool -> Std.BitVec 1
+ | true   => 1
+ | false  => 0
```

Comment:
This change is orthogonal. I suggest to commit such trivial changes directly or push them into separate PRs. They just add noise to PRs.

Reply:
These were not me 😄 I thought it was you/Alex who made these changes when changing this PR to work with `Std`'s bitvec!

Reply:
I pushed this in a separate PR.